x-pack/plugin/src/test/resources/rest-api-spec/api/ml.update_data_frame_analytics.json

```diff
@@ -0,0 +1,29 @@
+{
+  "ml.update_data_frame_analytics":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/update-df-analytics.html",
```

Comment:
It looks like the docs url already publish on elastic.co is different:
```diff
- https://www.elastic.co/guide/en/elasticsearch/reference/current/update-df-analytics.html
+ https://www.elastic.co/guide/en/elasticsearch/reference/current/update-dfanalytics.html
```

Reply:
Ah, well spotted. I'll fix it.